chains/astar/server/Cargo.toml

```diff
@@ -11,16 +11,18 @@ anyhow = "1.0.69"
 async-std = { version = "1.12.0", features = ["tokio1"] }
 async-trait = "0.1.66"
 ethers = "2.0.7"
+futures = { version = "0.3", default-features = false, features = ["std"] }
 hex = "0.4.3"
+log = "0.4.17"
 parity-scale-codec = "3.4.0"
 rosetta-config-astar = { version = "0.4.0", path = "../config" }
 rosetta-config-ethereum = { version = "0.4.0", path = "../../ethereum/config" }
 rosetta-server = { version = "0.4.0", path = "../../../rosetta-server" }
 rosetta-server-ethereum = { version = "0.4.0", path = "../../ethereum/server" }
-serde_json = "1.0.94"
+serde_json = "1.0"
```

Comment:
I removed the minor tag from some libs because it allows cargo to solve and get the latest minor version, so we don't have to update it manually every time.